native/cocos/renderer/gfx-validator/CommandBufferValidator.cpp

```diff
@@ -388,7 +388,7 @@ void CommandBufferValidator::draw(const DrawInfo &info) {
         if (!_curStates.descriptorSets[i]) continue; // there may be inactive sets
         const auto &dsBindings = _curStates.descriptorSets[i]->getLayout()->getBindings();
         const auto &psoBindings = psoLayouts[i]->getBindings();
-        CC_ASSERT(psoBindings.size() == dsBindings.size());
+        // CC_ASSERT(psoBindings.size() == dsBindings.size());
```

Comment:
Is builtin per-pass binding of [0: CCCamera , 1: CCConst] failed this validation?

Reply:
Need to roll back here.